mob.go

```diff
@@ -276,10 +278,10 @@ func main() {
 	execute(command, parameters, configuration)
 }
 
-func hasNoCommits() bool {
-	_, _, err := runCommandSilent("git", "rev-list", "-n", "1", "HEAD")
-	return err != nil
-
+func hasCommits() bool {
+	commitCount := silentgit("rev-list", "--all", "--count")
+	println(commitCount)
```

Comment:
Do we really like to print that? I don‘t think so. And if we would like to print it we should use the say function.

Reply:
its a mistake, thanks

Reply:
fixed